### ISSUE TITLE
RP mode capable?

### ISSUE THREAD

**Issue by chrisSimplyHome** [kevinohara80/elkington#7]
Is this software capable of putting an Elk M1 into Remote Programming mode? Specifically, connecting to the ElkRP2 software. After looking through the code, I don't think so, but I'm not very familiar with JS so I could've easily overlooked something.

**Comment by kevinohara80** [comment]
I don't think so but if it's in the RS232 protocol I could add it. Wanna check?

**Comment by chrisSimplyHome** [comment]
I haven't seen it in any of the protocol documents from Elk, and it seems as though there is some specific handshaking going on between the M1 and ElkRP2 that seems unlikely to be documented in generic RS232 protocol. Thanks anyway!
